server/views/pages/create-goal.njk

```diff
@@ -70,110 +63,161 @@
+            },
+            {
+                divider: "or"
+            },
+            {
+                value: "custom",
+                text: locale.dateSelection.customRadioOptionText,
+                conditional: {
+                    html: customDateHtml
+                }
+            }
+        ]
+    }) }}
+{% endset -%}
 
 {% block content %}
     <div class="govuk-width-container">
         <a href="#" class="govuk-back-link">Back</a>
         <main class="govuk-main-wrapper">
             <div class="govuk-grid-row">
+                <div class="govuk-grid-column-one-third">
+                    {{ mojSideNavigation({
+                      label: 'Side navigation',
+                      items: data.navigationLinks
+                    }) }}
+                </div>
                 <div class="govuk-grid-column-two-thirds">
-                    <span class="govuk-caption-xl">{{ locale.section | replace("{AREA_OF_NEED}", data.referenceData.name) }}</span>
-                    <h1 class="govuk-heading-xl">{{ locale.title | replace("{POP_NAME}", data.popData.firstName) }}</h1>
-                    <div class="govuk-inset-text">
-                        {# todo: Probably needs to be changed to be an auto-generated array of HTML for notes #}
-                        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">{{ locale.notes.title | replace("{POP_NAME}", data.popData.firstName) }}</h2>
-                        <p class="govuk-body govuk-!-font-size-16">{{ locale.notes.attribution
-                            | replace("{NOTE_CREATION_DATE}", (data.noteData.creationDate | formatSimpleDate ))
-                            | replace("{NOTE_AUTHOR}", (data.noteData.author ))
-                        }}</p>
-                        <p class="govuk-body">{{ data.noteData.content }}</p>
-                    </div>
+
+                    <span class="govuk-caption-l">{{ data.displayAreaOfNeed }}</span>
+                    <h1 class="govuk-heading-l">{{ locale.title | replace("{POP_NAME}", data.popData.givenName) }}</h1>
 
                     <form id="create-goal-form" method="POST">
                         <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-                        <input type="hidden" name="area-of-need" value="{{ data.referenceData.name }}">
+                        <input type="hidden" id="_areaOfNeed" value="{{ data.areaOfNeed }}">
+                        <input type="hidden" name="area-of-need" value="{{ data.displayAreaOfNeed }}">
+                        <input type="hidden" id="page" value="goals">
+                        <div class="govuk-form-group">
+                            <h1 class="govuk-label-wrapper">
+                                <label class='govuk-label--m' for="step-name">What goal should {{ data.popData.firstName }} try to achieve?</label>
```

Comment:
Should this be `givenName` ? When I run it up locally all usages of firstName render as empty or undefined.